### ISSUE TITLE
Example commands for add_command and remove_command

### ISSUE THREAD

**Issue by ehsanminachi** [nanopack/yoke#17]
Hello,

Would you please post a working config file example, specially for `add_command` and `remove_command`

Thank you.

**Comment by tylerflint** [comment]
@ehsanminachi Yes, in fact we're in the process of creating a github pages site with documentation. Hopefully in the coming weeks we should have yoke fully documented with usage examples.

Thanks!
